rmw_zenoh_cpp/src/detail/liveliness_utils.hpp

```diff
@@ -23,6 +23,9 @@
 #include <string>
 #include <vector>
 
+#include "logging_macros.hpp"
+
+#include "rmw/error_handling.h"
```

Comment:
Both of these can be moved back into `liveliness_utils.cpp`.

Reply:
oops my bad https://github.com/ros2/rmw_zenoh/pull/266/commits/dd4498391084a774d79f2271b68d87154ec55f4e